KP5-Topic2/KP4-Topic2/target/m2e-wtp/web-resources/META-INF/maven/com.aguirre/KP4-Topic2/pom.xml

```diff
@@ -0,0 +1,21 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Please include the folder target in .gitignore file to avoid to upload to the repo these generated files